Authentication_and_Authorization/hello-security/login-controller.php

```diff
@@ -6,19 +6,47 @@
 <body>
 <a href="index.php"> Take me home, country roads ... </a> <br>
 <?php
+
 include "db-connection.php";
+
+function getHashedPassword($conn, $email){
+
+    $sqlQueryForPassword = "SELECT password FROM person WHERE email = :email;";
+
+    $getHashedPassword = $conn->prepare($sqlQueryForPassword);
+    $getHashedPassword->bindParam(':email', $email);
+    $getHashedPassword->execute() or die ("Not valid credentials. There is no user with this email.");
+
+    $result = $getHashedPassword->fetch(PDO::FETCH_BOTH);
+    $hashedPassword = $result['password'];
```

Comment:
Php error when trying to log in with non existing user: 
![image](https://user-images.githubusercontent.com/10218337/80319624-14efab80-881a-11ea-8f1e-6df23d4e65fc.png)